### DISCUSSION TITLE
Nonlinear solve did not converge due to DIVERGED_FNORM_NAN

### DISCUSSION THREAD

**Discussion by anupkgeos** [idaholab/moose#26103] [Q&A Modules: Porous Flow] [answered]
Hi all,
I am trying to simulate a hydromechanical problem, it runs okay but when I add initial in-situ stress then it doesn't converge
```
Time Step 1, time = 10, dt = 10
Pressure 86664.6 is outside the range of tabulated pressure (100000, 1e+08).
Nonlinear solve did not converge due to DIVERGED_FNORM_NAN iterations 6
```
![image](https://github.com/idaholab/moose/assets/145852108/0379de99-6b84-4434-bc5a-7e18a8eb4166)

In-situ stress is implemented as:
```
  [sxx0]
    type = ParsedFunction
    expression = '-0.7*${gravity} * y * (${solid_density} - ${fluid_density}) * (1.0 - ${porosity0})'  # initial effective stress that should result from weight force
  []
  [syy0]
    type = ParsedFunction
    expression = '-${gravity} * y * (${solid_density} - ${fluid_density}) * (1.0 - ${porosity0})'  # initial effective stress that should result from weight force
  []


   [ini_strain]
     type = ComputeEigenstrainFromInitialStress
     initial_stress = 'sxx0 0 0  0 syy0 0  0 0 sxx0'
     eigenstrain_name = 'ini_strain'
   []
```
Please find my [input file here](https://github.com/anupkgeos/co2injection).  I don't know how to fix it. Any help is appreciated. Thank you!

**Comment by lindsayad** [comment, marked as answer]
This error message

> Pressure 86664.6 is outside the range of tabulated pressure (100000, 1e+08).

seems pretty clear. I would try to fix that

**Reply by anupkgeos** [reply]
I fixed the issue. It was due to the wrong(low) initial stress.
